github.go

```diff
@@ -287,12 +287,15 @@ func (m *GithubClient) GetPullRequest(prNumber, commitRef string) (*PullRequest,
 			PullRequest struct {
 				PullRequestObject
 				Commits struct {
-					Edges []struct {
-						Node struct {
-							Commit CommitObject
-						}
+					Nodes []struct {
```

Comment:
This tweak was to standardise on using `Nodes` rather than `Edges`, as we're only concerned with the data itself...
> https://johnkavanagh.co.uk/writing/gatsby-and-graphql-nodes-vs-edges/

Reply:
👍